samples/net/sockets/big_http_download/prj.conf

```diff
@@ -3,6 +3,7 @@ CONFIG_NEWLIB_LIBC=y
 CONFIG_MBEDTLS=y
 CONFIG_MBEDTLS_ENABLE_HEAP=y
 CONFIG_MAIN_STACK_SIZE=2176
+CONFIG_SAMPLE_BIG_HTTP_DL_NUM_ITER=3
```

Comment:
Form @pfalcon in #26719:
Please don't change existing settings without good motivation. Such changes and motivation should go into a separate PR (on its own). Current usecase for big_http_download is that a user can start it and use for assessing IP stack overall stability, e.g. wait and watch until it downloads 1GB total data (then one kinda assume that "IP stack in Zephyr works").

Reply:
@pfalcon The value `CONFIG_SAMPLE_BIG_HTTP_DL_NUM_ITER` could not be find in any KConfig in the original implementation of the application! Do you mean, that this was expected?

Reply:
> The value `CONFIG_SAMPLE_BIG_HTTP_DL_NUM_ITER` could not be find in any KConfig in the original implementation of the application!

Hmm, this config option is found in `samples/net/sockets/big_http_download/Kconfig`.
Not sure what is the issue here, difficult to follow the discussion as you closed the original PR and created a new one. In the future, please use the original PR when submitting changes (unless the PR is too convoluted or messy), that way the discussion is easier to follow.

Reply:
Sorry for closing previous PR. It was not clear for me how to change PR name and branch's name. Next time I will be better :)

You'r right about `samples/net/sockets/big_http_download/Kconfig` and  `CONFIG_SAMPLE_BIG_HTTP_DL_NUM_ITER `. Thus the commit `samples: net: sockets: big_http_download: preparing upgrade` with corresponding change will be dropped within next push.